Makefile

```diff
@@ -5,6 +5,17 @@ CARGO = cargo
 # Let's be very explicit about it for now.
 TESTS_DIR := `pwd`/tests
 TEST_RESULTS_DIR ?= workspace/test-results
+
+# In order to be consumed by the ETE Test Metric Pipeline, files need to follow a strict naming
+# convention: {job_number}__{utc_epoch_datetime}__{workflow}__{test_suite}__results{-index}.xml
```

Comment:
I don't see `CIRCLE_PROJECT_REPONAME` here, is that considered part of workflow (and is `-index` is considered optional)?

Reply:
`CIRCLE_PROJECT_REPONAME` is one of a set of standardized environment variables that CircleCI provides, list --> https://circleci.com/docs/variables/

And yes `-index` is optional, more here --> https://mozilla.github.io/ecosystem-test-scripts/reference-guides/project_onboarding_procedure.html#1-setup-cicd-to-push-test-result-and-coverage-data-to-gcs

Reply:
I meant within: `{job_number}__{utc_epoch_datetime}__{workflow}__{test_suite}__results{-index}.xml` -- it looks like we're producing a `{utc_epoch_datetime}__{reponame}__{workflow}` in the prefix below (unless we're considering `reponame` as part of `workflow`)

Reply:
Oh nevermind, the docs linked make it clear it is in fact part of `workflow` (thanks, I was looking for such docs)

Reply:
Sry I misunderstood! But yay for documentation!